components/PyTorch/pytorch-kfp-components/LICENSE

```diff
@@ -0,0 +1,190 @@
+                                 Apache License
```

Comment:
Now this license is the same as root license. Do you want to keep a copy here?

(No opinion on this, just confirming)

Reply:
Yes, this will be preferred from our side to make things explicit